.github/workflows/examples-esp32.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
                    .environment/gn_out/.ninja_log
                    .environment/pigweed-venv/*.log
             - name: Build example All Clusters App
-              timeout-minutes: 15
+              timeout-minutes: 20
```

Comment:
Why do we need to change/increase the timeout?

Reply:
Because the CI always fails with timeout.